06-Modelling-Rigid-Bodies/01-Introduction-to-Rigid-Bodies.ipynb

```diff
@@ -0,0 +1,840 @@
+{
```

Comment:
<p>Clarify that the dimer positions are relative to (0,0,0) (i.e. that the central particle is assumed to be the origin of the coordinate system).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>